### PR TITLE
Kevin/Update Regulatory - kid pdf with eu language compatibility

### PR DESCRIPTION
**Pull request by kevinw-binary**
# Description

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/regulatory/_document_accordion.js [hunk 1]
```diff
@@ -140,6 +141,16 @@ const DocumentAccordion = () => {
...
     }
 
+    const selected_language = locale.locale.language || 'en'
+    const supported_languages = ['es', 'it', 'pl', 'pt']
+
+    const is_supported_language = (language) => {
+        if (supported_languages.includes(language)) {
+            return true
+        }
+        return false
```
Comment (AbdallaMourad) [inline src/pages/regulatory/_document_accordion.js:151]:
```suggestion
    const is_supported_language = (language) => supported_languages.includes(language)
```

File: src/pages/regulatory/_document_accordion.js [hunk 2]
```diff
@@ -180,48 +191,23 @@ const DocumentAccordion = () => {
...
+                            <FlexText
+                                key={idx}
+                                color="red"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                href={`/regulatory/kid/${
+                                    is_supported_language(selected_language)
+                                        ? selected_language + '/' + data.ref
+                                        : data.ref
+                                }`}
```
Comment (AbdallaMourad) [inline src/pages/regulatory/_document_accordion.js:204]:
```suggestion
                                href={`/regulatory/kid/${
                                    is_supported_language(selected_language)
                                        ? selected_language + '/'
                                        : ''
                                }data.ref`}
```

**Review by AbdallaMourad — APPROVED** [review]

**Review by yashim-deriv — APPROVED** [review]

File: src/pages/regulatory/_document_accordion.js [hunk 3]
```diff
@@ -122,7 +123,7 @@ const RTS27_28 = () => (
     </>
 )
 
-const DocumentAccordion = () => {
+const DocumentAccordion = (locale) => {
```
Comment (mustofa-binary) [inline src/pages/regulatory/_document_accordion.js:126]:
```suggestion
const DocumentAccordion = ({ locale }) => {
```

File: src/pages/regulatory/_document_accordion.js [hunk 4]
```diff
@@ -140,6 +141,11 @@ const DocumentAccordion = () => {
         width: '100%',
     }
 
+    const selected_language = locale.locale.language || 'en'
```
Comment (mustofa-binary) [inline src/pages/regulatory/_document_accordion.js:144]:
```suggestion
    const selected_language =locale.language || 'en'
```

File: src/pages/regulatory/_document_accordion.js [hunk 5]
```diff
@@ -180,48 +186,23 @@ const DocumentAccordion = () => {
...
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                href={`/regulatory/kid/${
+                                    is_supported_language(selected_language)
+                                        ? selected_language + '/'
+                                        : ''
+                                }${data.ref}`}
+                                m="1.6rem 2.4rem"
+                            >
+                                <img src={PDFIcon} alt="pdf icon black" />
```
Comment (mustofa-binary) [inline src/pages/regulatory/_document_accordion.js:202]:
```suggestion
                                <img src={PDFIcon} alt={data.title} />
```

File: src/pages/regulatory/index.js [hunk 6]
```diff
@@ -104,7 +105,7 @@ const ExternalBoldLink = styled(LocalizedLink)`
 const Content = styled.div`
     display: contents;
 `
-const Regulatory = () => {
+const Regulatory = (locale) => {
```
Comment (mustofa-binary) [inline src/pages/regulatory/index.js:108]: same here

**Review by mustofa-binary — APPROVED** [review]
Please check out comment for future refactoring

**Pull request merged**